### PR TITLE
Updated addresses for sr25519

### PR DESCRIPTION
**Pull request by JoshOrndorff**
This is my attempt to solve https://github.com/polkadot-js/apps/issues/828 .

### CODE REVIEW

File: packages/app-js/src/snippets/transfer.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 export default `// transfer
-const sender = '5GoKvZWG5ZPYL1WUovuHW3zJBWBP5eT8CbqjdRY4Q6iMaDtZ';
+const sender = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKv3gB';
 const recipient = '5GTZEKgRKX2a5hepNjwTBzR5nR6UqtEsMZ4TnL1Yuah1N2Tu';
```
Comment (JoshOrndorff) [inline packages/app-js/src/snippets/transfer.ts]: I wasn't sure there this recipient address came from.

Comment (JoshOrndorff) [inline packages/app-js/src/snippets/transfer.ts]: I wasn't sure there this recipient address came from.

Comment (jacogr) [reply packages/app-js/src/snippets/transfer.ts]: Think it was just a random sample address, @Stefie ? I suppose we could have just created a random address ourselves with something like, but probably overkill as a sample -

```js
const { randomAsU8a } = hashing;

...
// a random recipient address, from 32-byte public key
const recipient = keyring.encodeAddress(randomAsU8a());
...
```

**Comment by JoshOrndorff** [comment]
Looks like I can't merge it myself.

File: packages/app-js/src/snippets/transfer.ts (hunk 1 above)
Comment (Stefie) [reply packages/app-js/src/snippets/transfer.ts]: @jacogr You added this example when you added the Signer interface, so if you don't know it I think we can change it to sth. random or use BOB

**Pull request merged**